Add notes for 2019.12.03 (#8)
p2pcommons/working-group: 20191203.md

-   Attending
    - Julian Gruber [@juliangruber](https://twitter.com/juliangruber)
    - Diego Paez [@carax](https://twitter.com/carax)
    - Chris Hartgerink [@chartgerink](https://twitter.com/chartgerink)
-   [for reference] [Versioning scheme](https://drive.google.com/a/libscie.org/open?id=1Hp4gIUVNePO33lGcYeK8dOwY6EZWCQrL7XrURIILslg)
-   General announcements
    - Roadmaps for each project are/will be in the GitHub wiki. We will discuss those over time anyway, but then you know where to find them if you need them.
    - Hypergraph Registry --> Hypergraph Vault
-   [`@p2pcommons/specs`](https://github.com/p2pcommons/specs)
    - Update by Chris
      - Keep the text spec open over the next weeks, because slow and steady improvements are best right now
      - Added an HTTPS recommendation
      - Next up? Dogfooding interoperability spec

>         - JG: Not that urgent. We can always add code to the spec?

Reviewer: ```suggestion
        - JG: Not that urgent. We can eventually add code to the spec?
```

        - CHJH: We can always give a bit of money for alternative p2pcommons applications to proxy dogfood it
    - Questions
-   [`@p2pcommons/sdk-js`](https://github.com/p2pcommons/sdk-js)
    - Update by Diego 
      - Continued working on the networking part of `sdk-js`
      - Worked on realistic scenario for reproducing content
        - Got a connection to swarm and debugged further
        - Small scenario of the problem
          - Seed archive and download from other place using corestore-swarm
          - mirror-folder worked, couldn't find problem
          - switched to hyperswarm
          - made some changes to have only one swarm for multiple hyperdrives, created this manually
        - went on a new debugging session, showed that there was a primary problem with versions

>           - last version of hyperprotocol which isn't in hyperdrive

Reviewer 2: ```suggestion
          - `corestore-swarm-networing` was using the latest version (v7+) of `hypercore-protocol` which isn't same that hyperdrive (v10+) is using (v6+).  
```

>           - manual install of hyperprotocol v6 fixed it

Reviewer 2: ```suggestion
          - manual install of `hypercore-protocol` v6 fixed it
```
